packages/app/src/index.css

```diff
@@ -928,3 +928,10 @@ svg.zap-solid {
 .hide-scrollbar::-webkit-scrollbar {
   display: none; /* Safari and Chrome */
 }
+
+@media (min-width: 768px) {
+  .rt-border{
+    border-right-width: 1px;
+    border-right-color: red;
```

Comment:
I dont think it was supposed to be red, thats just for emphasis it should be `var(--border-color)` probably